src/components/ContractDeploy/ContractDeploy.tsx

```diff
@@ -26,12 +26,14 @@ interface IState {
   gasSupply: number;
   error: Error | null;
   deployed: object;
-  methodName: string;
+  methodName: string | null;
```

Comment:
A  `null` check is required now in the component where ever `methodName` is used from `state`. I have added it in `handleMethodnameInput`. Do the same if any.